src/pudl/metadata/resources/eia861.py

```diff
@@ -599,6 +736,12 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_utility_data_misc": {
```

Comment:
A table rename that suggests itself to me here is `core_eia861__yearly_utility_activities`.

Maybe:
```
"""A table of boolean values indicating what kind of business activities each utility engages in.

For example, do they provide transmission, distribution, or generation services? Wholesale or retail marketing? Do they have an alternative fuel vehicle program?
"""
```

(there are a couple of typos in the current description, "includeing" and "bundeled")

Reply:
I like this description, but it's not about whether the utility has alternative fuel vehicle programs, the form actually says "Did Your Company Operate Alternative-Fueled Vehicles During the Year?"

Reply:
To be addressed in #3301